From dce462816cc7768a076b0023b9f730ba111db38b Mon Sep 17 00:00:00 2001
From: Alex Marginean <alexandru.marginean@nxp.com>
Date: Mon, 10 Sep 2018 18:00:58 +0300
Subject: [PATCH 297/706] enetc: fix driver dependency

remove MDIO dependency, we do not call MDIO directly and fixed link works
just as good without it.
add PCI_MSI dependency, we actually call that and crash without it.

Signed-off-by: Alex Marginean <alexandru.marginean@nxp.com>
(cherry picked from commit 9f20cf66f78afc8ec023448c1dd4422ea1f9f113)
(cherry picked from commit bfcb586b3cd566885ff26ce34a9e69fd2140de07)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/freescale/enetc/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/freescale/enetc/Kconfig b/drivers/net/ethernet/freescale/enetc/Kconfig
index a83eb62652c1..2f8a1766ad62 100644
--- a/drivers/net/ethernet/freescale/enetc/Kconfig
+++ b/drivers/net/ethernet/freescale/enetc/Kconfig
@@ -1,12 +1,12 @@
 config FSL_ENETC
 	tristate "ENETC PF driver"
-	depends on PCI && FSL_XGMAC_MDIO && FSL_ENETC_MDIO
+	depends on PCI && PCI_MSI
 	---help---
 	  TBD
 
 config FSL_ENETC_VF
 	tristate "ENETC VF driver"
-	depends on PCI
+	depends on PCI && PCI_MSI
 	---help---
 	  TBD
 
-- 
2.17.1

